extension/docs/DevelopmentAndDeploymentGuide.md

```diff
@@ -36,6 +36,7 @@ Environment variables to configure the notification module:
 |`API_EXTENSION_BASE_URL` | URL of the Extension module server. In case of any payment changes, [CTP API extension](https://docs.commercetools.com/http-api-projects-api-extensions) will call this URL and pass the payment object in body. | YES | |
 |`LOG_LEVEL` | bunyan log level (`trace`, `debug`, `info`, `warn`, `error`, `fatal`)| NO | `info`
 |`PORT` | port on which the application will run | NO | 8080
+|`DISABLE_ENSURE_RESOURCES` | disables the creation of resources in commercetools (e.g. custom types) | NO | `false`
```

Comment:
Negations in variable names are confusing to read once you need to negate those in `if` statements. How about:
`ENSURE_RESOURCES` with default `true`?

Reply:
I think it makes it more explicit that you are specifically disabling the ensure resources portion of the application.

Reply:
I get the point but it is just not a good practice in general. Just some examples:
http://truelogic.org/wordpress/2009/02/27/how-not-to-name-a-boolean-variable/
http://davej.io/2018/05/negative-naming.html
https://softwareengineering.stackexchange.com/questions/196830/boolean-method-naming-affirmative-vs-negative

Reply:
Have changed this to `ENSURE_RESOURCES` now with default of `true`

Reply:
Thanks!